helpers.go

```diff
@@ -0,0 +1,24 @@
+package realis
+
+import (
+	"context"
+
+	"github.com/paypal/gorealis/gen-go/apache/aurora"
+)
+
+func (r *realisClient) jobExists(key aurora.JobKey) bool {
+	resp, err := r.client.GetConfigSummary(context.TODO(), &key)
+	if err != nil {
+		return false
```

Comment:
errr != nil does not mean that the job does not exist. 
We should print out error here.

Reply:
Yeah, we can return an error here